src/components/TertiaryButton/TertiaryButton.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { TertiaryButtonWrapper } from './TertiaryButton.styles';
+
+const TertiaryButton = ({ content, onClick, type = 'button' }) => {
+  return (
+    <TertiaryButtonWrapper type={type} onClick={onClick}>
+      {content}
+    </TertiaryButtonWrapper>
+  );
+};
+
+TertiaryButton.propTypes = {
+  content: PropTypes.string.isRequired,
+  type: PropTypes.oneOf(['submit', 'button', 'reset']),
+  onClick: PropTypes.func
```

Comment:
I asked @ammersive about this as well, what do you both think about having the `onClick` handler prop being required on buttons? That way there's less likelihood of a bug with a button not performing an action.

Happy with it either way we decide, just a thought.

Reply:
Scratch that, not needed. Here was Rob's response:

> No, you don't necessarily need one. In forms for example, the event is handled my the submission of the form as opposed to the button so a onClick is not always required.

Reply:
frankly i literally copied your tests for the primary button, I'm still studying them in my own time and not entirely comfortable as to how and when to do them, So I'm glad Rob said no need :P